t/data_migration_pipeline/main.go

```diff
@@ -23,17 +23,22 @@ import (
 	"strings"
 
 	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/metrics"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/io/spannerio"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/io/textio"
 	"github.com/apache/beam/sdks/v2/go/pkg/beam/runners/direct"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/x/beamx"
 )
 
 var (
 	input    = flag.String("input-csv-path", "", "The path of the input MySQL CSV dumped .")
 	database = flag.String("spanner-database", "", "The path of the output Spanner database.")
 	table    = flag.String("spanner-table", "", "The name of the output Spanner table.")
+	dryRun   = flag.Bool("dry-run", true, "whether the specified run is a dry run")
```

Comment:
This feels odd to me to default a dry run to true.

In general prefer default false for boolean flags: https://github.com/abcxyz/readability/wiki/Go-Best-Practices#boolean-flags

Reply:
Done